### PR TITLE
Feature/add task wait

### PR DESCRIPTION
**Pull request by Yiming1997**
add taskWait to wait all task done

### CODE REVIEW

**Comment by Yiming1997** [comment]
@0x9n0p pls review ,thank you:)

File: pool.go [hunk 1]
```diff
@@ -22,9 +23,14 @@ func Open(ctx context.Context, config Config) *Pool {
 }
 
 func (p *Pool) SendTask(task Task) {
+	p.wg.Add(1)
 	p.pipe <- task
 }
 
+func (p *Pool) TaskWait() {
```
Comment (0x9n0p) [inline pool.go:30]: Why do we need TaskWait?

**Comment by 0x9n0p** [comment]
@Yiming1997  Why don't you handle the wait group outside the library? It would give you more control and allow you to have simpler APIs.

**Comment by Yiming1997** [comment]
@0x9n0p I got you..but i think we should have some functions built-in to wait tasks done, I will give another new solution later,now closing.thank you

**Pull request closed without merging**